src/commands/publish/mod.rs

```diff
@@ -61,6 +61,15 @@ fn publish_script(user: &User, name: &str) -> Result<(), failure::Error> {
                 .body(build_js_script()?)
                 .send()?
         }
+        ProjectType::Webpack => {
```

Comment:
Do you see `ProjectType::Webpack` replacing `ProjectType:: JavaScript `? I guess we're keeping both until we reach feature parity (ie for the preview). Was just wondering how you see this moving forward and evolving in the longer term

Reply:
In discussion #102

Reply:
there should always be an option to not use webpack.